src/index.html

```diff
@@ -6,8 +6,48 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <section class="card" data-qa="card">
+      <div class="card__img">
+        <img
+          class="card__img-link"
+          src="./images/imac.jpeg"
+          alt="imac"
+        >
+      </div>
+
+      <h1 class="card__tittle">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__reviews">
+        <div class="stars">
```

Comment:
```suggestion
        <div class="stars stars--4">
```
The number of active stars should depend on the modifier